<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>borrow_return_book</class>
 <widget class="QDialog" name="borrow_return_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog{
    background-image: url(&quot;:/Images/ylswjsy7stw-2048x1201.jpg&quot;);
    background-repeat: no-repeat;
    background-position: center;
    background-attachment: fixed;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>-70</y>
     <width>711</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
background:transparent;
}



QLabel {
    font-family: &quot;Segoe UI&quot;;       /* Or any font you like */
    font-size: 16px;               /* Adjust size as needed */
    font-weight: bold;             /* This makes it bold */
    color: white;                  /* Optional: sets text color */
}
QLineEdit {
    color: #FFFFFF;  /* White text color */
    background: transparent;
    border: none;
    border-bottom: 2px solid white;
    padding: 2px;
    font-weight: bold;
}

QLineEdit:focus {
    border-bottom: 2px solid #FFD700; /* gold underline when focused */
}




</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Fines</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>491</width>
      <height>571</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="borrow_book_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>510</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrow Book</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>List of Books</string>
     </property>
    </widget>
    <widget class="QListView" name="book_list">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>441</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView{
background:rgba(255, 255,255, .8);
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="fines_money">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>300</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelback">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background:rgba(0,0,0,200);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="return_book_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Return Book</string>
    </property>
   </widget>
   <widget class="QComboBox" name="borrowed_books_combo_box">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <zorder>labelback</zorder>
   <zorder>label</zorder>
   <zorder>groupBox</zorder>
   <zorder>fines_money</zorder>
   <zorder>return_book_button</zorder>
   <zorder>borrowed_books_combo_box</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
